docs/tools/terraform/howto/use-opentofu.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Use OpenTofu with Aiven Provider for Terraform
+sidebar_label: Use OpenTofu
+---
+
+[OpenTofu](https://opentofu.org/) is an open source infrastructure as code tool that you can use to configure your Aiven infrastructure.
+
+Set up your first Aiven project and service using this example to get started with
+OpenTofu. If you already use the [Aiven Provider for Terraform](/docs/tools/terraform.md),
+you can [migrate your Terraform resources to OpenTofu](https://opentofu.org/docs/intro/migration/).
+
+## Prerequisites
+
+-   [Sign up for
+    Aiven](https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=devportal&utm_content=repo)
+-   [Install OpenTofu](https://opentofu.org/docs/intro/install/)
+-   [Create an authentication
+    token](https://docs.aiven.io/docs/platform/howto/create_authentication_token.html)
+
+## Configure your project and services
+
+Set up the OpenTofu project in an empty folder:
+
+1.  Create a new file, `provider.tf`, and add the following code to declare a dependency
+    on the Aiven Provider, specifying the
+    [version](https://registry.terraform.io/providers/aiven/aiven/latest).
+
+    ```terraform
+    terraform {
+      required_providers {
+        aiven = {
+          source  = "aiven/aiven"
+          version = ">=4.0.0, < 5.0.0"
+        }
+      }
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_token
+    }
+    ```
+
+1. Create a file named `project.tf` and add the following code to create an Aiven project
+   in your organization:
+
+   ```terraform
+    # Get information about your organization
+    data "aiven_organization" "main" {
+      name = "ORGANIZATION_NAME"
+    }
+    # Create a new project in your organization
+    resource "aiven_project" "example_project" {
+      project    = "ORGANIZATION_NAME-example-project"
+      parent_id = data.aiven_organization.main.id
+    }
+   ```
+
+   Where `ORGANIZATION_NAME` is your
+   [Aiven organization](/docs/platform/concepts/projects_accounts_access) name.
+
```

Comment:
This paragraph breaks the list. I suppose it's because of the 1 space indentation for this item vs the 2 space indent for the rest of the list?